base/linalg/triangular.jl

```diff
@@ -1087,15 +1174,116 @@ function logm{T<:Union{Float64,Complex{Float64}}}(A0::UpperTriangular{T})
          1.276404810806775e-001,
          2.060962623452836e-001,
          2.879093714241194e-001]
+
```

Comment:
The `i`-th entry is the maximal value of `||X||` such that `||X||<1` and a complicate expression involving `X` and `i` is smaller than our current `eps`. I have no idea how long it would take to compute them for a given precision, but it might be matter of minutes, that's why I suggested to store them hard coded.
I'll try to find out how these values can be computed and fix the other issues you pointed out in the meanwhile.